### PR TITLE
BinOp order

### LINKED ISSUE #14: Parser doesn't seem to create correct BinOp nodes
The AST defines them as having the operator at the end: 

[`BinOpNode = Node.new_type(:binop, :left, :right, :op)`](https://github.com/sgnr/Liquider/blob/f6cdbde7d8bde6aa090c6d50129af19bf3d733ca/lib/liquider/ast.rb#L46)

The [parser tests](https://github.com/sgnr/Liquider/blob/f6cdbde7d8bde6aa090c6d50129af19bf3d733ca/spec/parser_spec.rb#L33-L37) puts it in the "left" property:

```ruby
        Ast::BinOpNode.new(
          :*,
          Ast::StringNode.new('bar'),
          Ast::SymbolNode.new('baz'),
        ),
```

### PR DESCRIPTION
**Pull request by gmalette**
Fixes #14

### CODE REVIEW

**Comment by gmalette** [comment]
ping @sgnr

File: lib/liquider/liquid.y [hunk 1]
```diff
@@ -73,7 +73,7 @@ rule
   AdditiveExpression
   : MultiplicativeExpression
   | AdditiveExpression PLUS MultiplicativeExpression { result = Ast::BinOpNode.new(:+, val[0], val[2]) }
-  | AdditiveExpression MINUS MultiplicativeExpression { result = Ast::BinOpNode.new(:-, val[0], val[2], :-) }
```
Comment (sgnr) [inline lib/liquider/liquid.y]: did this even get reviewed lol hack days.

**Comment by sgnr** [comment]
Test failures, you'll need to rebase.

File: lib/liquider/liquid.y (hunk 1 above)
Comment (gmalette) [inline lib/liquider/liquid.y]: The tests only test the values, so if `left` is `:+` on both sides, they're equal

**Comment by sgnr** [comment]
Otherwise :shipit:

**Pull request merged**